src/components/Board/index.js

```diff
@@ -40,15 +40,12 @@ function UncontrolledBoard({
   allowAddColumn,
   renderColumnAdder,
   onNewColumnConfirm,
-  onColumnRemove,
   renderColumnHeader,
   allowRemoveColumn,
   allowRenameColumn,
-  onColumnRename,
   onCardNew,
   renderCard,
   allowRemoveCard,
-  onCardRemove,
```

Comment:
The onColumnRemove, onColumnRename and onCardRemove callbacks are still needed, they just shouldn't be mandatory as they are currently, forcing us to specify them even if we don't need to do anything after these actions.

I made a [sandbox](https://codesandbox.io/s/issue-140-react-kanban-23chs) to better demonstrate the problem to you.